pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -157,7 +155,22 @@ func (c *controller) Get(typ, name, namespace string) *model.Config {
 		return nil
 	}
 
-	return nil
+	out := &model.Config{}
+	if typ == model.Gateway.Type {
+		*out = ConvertIngressV1alpha3(*ingress, c.domainSuffix)
```

Comment:
rename `ConvertIngressV1alpha3` to `ConvertIngress` since some of this logic/naming is still around from v1alpha2 rule logic